assets/source/setup-guide/app/helpers/connect-advertiser.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+const connectAdvertiser = async ( trackingAdvertiser, trackingTag ) => {
+	try {
+		const results = await apiFetch( {
+			path: `${ wcSettings.pinterest_for_woocommerce.apiRoute }/tagowner/`,
```

Comment:
Remove the trailing slash, so that we can see the API request better in browser dev tools' network panel.

```suggestion
			path: `${ wcSettings.pinterest_for_woocommerce.apiRoute }/tagowner`,
```

For reference, with the trailing slash, this is what we see in network panel:

![image](https://user-images.githubusercontent.com/417342/148742382-7bf74102-fa76-4170-ace5-ddffa304c616.png)